codeexamples/SYCLUsingCUDAStaticLibraryOfKernels/.gitignore

```diff
@@ -0,0 +1,3 @@
+*.o
```

Comment:
IMO it would make sense to put the object file ignore stuff at the top level. Github has a skeleton gitignore we could copy from for C++ projects

Reply:
Is this a request for a change or just an opinion? We can certainly do this at a later date when the repository is more populated.

Reply:
A .gitignore file exists at one level up in codeexamples which has object file ignore rules included (along with other rules). Done.